src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -5,11 +5,26 @@
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
```

Comment:
The class implementation should not begin with an empty line. Please remove the unnecessary empty lines at the start of the class definition.